src/neptune_tensorflow_keras/impl/__init__.py

```diff
@@ -137,12 +140,13 @@ def _log_metrics(self, logs, category: str, trigger: str):
             try:
                 if metric in ("batch", "size") or metric.startswith("val_"):
                     continue
-                logger[metric].log(value)
+                logger[metric].append(value)
             except NeptuneException:
                 pass
 
     def on_train_begin(self, logs=None):
-        self._model_logger["optimizer_config"] = self.model.optimizer.get_config()  # it is a dict
+        optimizer_config = self.model.optimizer.get_config()  # it is a dict
+        self._model_logger["optimizer_config"] = stringify_unsupported(optimizer_config)
```

Comment:
This is to check my understanding.

We are adding `stringify_unsupported` so that if this changes in future to a object that Neptune doesn't know, then this will be a loud error.